x-pack/platform/packages/shared/onechat/onechat-server/tools/handler.ts

```diff
@@ -64,4 +65,9 @@ export interface ToolHandlerContext {
    * Logger scoped to this execution
    */
   logger: Logger;
+  /**
+   * Data Views service scoped to the current user.
+   * Can be used to access and query Kibana data views.
+   */
+  dataViewsService: DataViewsService;
```

Comment:
That's [this thing](https://github.com/elastic/kibana/blob/8d07d410d5a71969c04a357d89fcbc62277829d8/src/platform/plugins/shared/data_views/common/data_views/data_views.ts#L342-L351) right?

Genuine question, I wonder if we should just expose the plain underlying interface, as you did, or if we should create our own interface with the subset of APIs it make sense to expose instead.

Not sure how protective we should be here.